docker/php/conf.d/formulize.ini

```diff
@@ -4,9 +4,6 @@ error_reporting=E_ALL
 ; Memory Limit
 memory_limit = 256M
 
-; XDebug
```

Comment:
As mentioned in the lead comment, this code removes the xDebug reference in the formulize.ini that is used in Docker. This appears to be unnecessary because the dockerfile that controls our PHP instance already includes/enables xDebug? With the declaration in formulize.ini in place, there is an error on PHP start up, and that shows up as an error in every code block. With the declaration removed, this extra start up error is not present, and xDebug functionality in Docker/VSCode still works like normal.

Reply:
This needs verification on another system besides my own!

Reply:
I've confirmed that when the container is built we do the following `RUN docker-php-ext-enable mysqli pdo pdo_mysql gd xdebug zip` which is functionally equivalent to adding `zend_extension=xdebug` in the .ini file. Additionally I have built a version of this image and used it with success in running xdebug. This should be safe to remove.